File: 6KYU/Is a number prime?/solution.cs
using System;
public static class Kata
{
  public static bool IsPrime(int n)
  {
     if(n < 2) return false;
    
     for(int i = 2; i <= Math.Sqrt(n); i++)
         if(n % i == 0)
           return false;
    
     return true;
  }
}

File: 6KYU/Is a number prime?/solution.rs
fn is_prime(x: i64) -> bool {
    if x < 2 {
        return false;
    }
    
    let upper = f64::sqrt(x as f64) as i64+1;
    for i in 2..upper {
        if x % i == 0 {
            return false;
        }
    }
    
    return true;
}